### PR TITLE
added support for sublimetext (stdin/filename handling)

### PR DESCRIPTION
**Pull request by iham**
same problem as with other flake8 packages in sublime ... just forgot to pr ...

### CODE REVIEW

File: flake8_pep3101.py [hunk 1]
```diff
@@ -14,10 +13,7 @@ def __init__(self, tree, filename):
         self.tree = tree
 
     def run(self):
-        if self.filename == 'stdin':
-            lines = pycodestyle.stdin_get_value()
-            tree = ast.parse(lines)
-        elif self.tree:
+        if self.filename is 'stdin':
```
Comment (gforcada) [inline flake8_pep3101.py:16]: @iham this should be ``self.filename == 'stdin'``, python's ``is`` keyword is for identity check:
```
>>> a = 3
>>> b = 3
>>> a is b
True
>>> id(a)
94616584712504
>>> id(b) 
94616584712504
>>> c = 343434343434344
>>> d = 343434343434344
>>> c is d
False
>>> id(c)
94616585066984
>>> id(d)
94616585067032

File: flake8_pep3101.py [hunk 2]
```diff
@@ -14,10 +13,7 @@ def __init__(self, tree, filename):
         self.tree = tree
 
     def run(self):
-        if self.filename == 'stdin':
-            lines = pycodestyle.stdin_get_value()
```
Comment (gforcada) [inline flake8_pep3101.py:18]: @iham now I'm confused, in all other flake8 plugins the fix for handling stdin input was to use this approach rather that the self.tree as you are doing here, should it be changed everywhere as well? 🤔

**Review by gforcada — CHANGES_REQUESTED** [review]

**Comment by gforcada** [comment]
@iham if you want to test for ``stdin`` here is an example test:  https://github.com/gforcada/flake8-builtins/blob/master/run_tests.py#L122-L136

**Comment by gforcada** [comment]
@iham thanks for all the pull requests!! It is greatly appreciated!! 🙇

**Comment by gforcada** [comment]
@iham friendly ping 😄

**Comment by unformatt** [comment]
Is this PR dead? Would love to use this in SublimeText.

**Comment by gforcada** [comment]
It might be, though I'm not using sublime, that this is already working 🤞🏾 could any of you give a try to the new __2.0.0__ release?

Re-open if it is still not working 😄

**Pull request closed without merging**